lib/Bio/VertRes/Config/Pipelines/Mapping.pm

```diff
@@ -36,7 +36,6 @@ has 'module'                => ( is => 'ro', isa => 'Str', default  => 'VertRes:
 has 'reference'             => ( is => 'ro', isa => 'Str', required => 1 );
 has 'reference_lookup_file' => ( is => 'ro', isa => 'Str', required => 1 );
 has 'toplevel_action'       => ( is => 'ro', isa => 'Str', default => '__VRTrack_Mapping__' );
-has 'toplevel_admin_approval_required'   => ( is => 'ro', isa => 'Bool', default => 1 );
```

Comment:
Sure you want to delete this?   It will then inherit from Bio::VertRes::Config::Pipelines::Common so should work, but means the variable is only ever assigned a default value, and is tested once in Bio::VertRes::Config::TopLevel, so the variable is redundant.

You could simplify things by removing the variable entirely and lines 25, 73..79, 81 and 82 in TopLevel.pm ?

Or (probably better) you could just set the default to 0 at this line?

Reply:
If we don't use the admin approval functionality and are unlikely to ever need to use it again in the future, I think it might be an idea to remove it all then.

I didn't set the default to 0 because I was relying on the inheritance and noticed that assembly and annotation pipelines also didn't override the variable. So I was aiming for consistency among the different pipelines. We could be explicit and set this variable to have a default of 0 for all pipelines? My personal preference is to use inheritance, or remove all references to the variable (if we know we won't need the admin approval functionality).

Reply:
Fair comment -- removing it all would be perfectly reasonable.  We can always revert it, if we find a need for admin approval again (which I very much doubt)